common/src/main/java/com/thekillerbunny/goofyplugin/mixin/FiguraAPIManagerMixin.java

```diff
@@ -23,7 +24,9 @@ public class FiguraAPIManagerMixin {
 
     static {
         WHITELISTED_CLASSES.add(GoofyAPI.class);
+        WHITELISTED_CLASSES.add(CollectionAPI.class);
```

Comment:
wait, don't we have a thing for this in the plugin code?

Reply:
no this is how you do it